### PR TITLE
bug 1441369. Allow setting of kibana memory limits

### PR DESCRIPTION
**Pull request by jcantrill**
(cherry picked from commit 25f9aa9decdf5bf121213a8cf9b44bb3c57a9b77)

### CODE REVIEW

**Comment by jcantrill** [comment]
Add DO NOT MERGE until there is a cross referenced ansible PR

**Comment by jcantrill** [comment]
Merge with https://github.com/openshift/origin-aggregated-logging/pull/396

File: deployer/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin:v1.5.0-alpha.2
+FROM openshift/origin:v1.5.0
```
Comment (richm) [inline deployer/Dockerfile]: ok, but we're not using the deployer in 1.5
next step should be to remove deployer/ and move anything needed into hack/

**Comment by jcantrill** [comment]
@richm we only need it for support of oc cluster up.

**Comment by jcantrill** [comment]
[test]

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**